app/src/main/java/com/kennedy/peter/bloquery/firebase/User.java

```diff
@@ -0,0 +1,15 @@
+package com.kennedy.peter.bloquery.firebase;
+
+public class User {
+    public String UID, userName, email;
```

Comment:
These should be declared separately, by convention. 
Also, prefer immutable value objects by declaring the fields `private final`. More about immutability can be found [here](http://www.yegor256.com/2014/06/09/objects-should-be-immutable.html)